src/features/auth/authSlice.js

```diff
@@ -18,6 +19,16 @@ export const loginUser = createAsyncThunk("auth/login", async(credentials, { rej
   }
 });
 
+export const verifyUser = createAsyncThunk("auth/verify", async(encodedToken, { rejectWithValue })=>{
+  try{
+    const {data} = await axios.post("/api/auth/verify", {encodedToken});
+    console.log(data);
```

Comment:
you can remove...this console if everything working fine

Reply:
Thanks for pointing it out, will remove it.